VideoLocker/src/main/java/org/edx/mobile/task/ReadThreadTask.java

```diff
@@ -0,0 +1,37 @@
+package org.edx.mobile.task;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import org.edx.mobile.discussion.DiscussionThread;
+import org.edx.mobile.discussion.DiscussionThreadUpdatedEvent;
+import org.edx.mobile.http.RetroHttpException;
+
+import de.greenrobot.event.EventBus;
+
+public class ReadThreadTask extends Task<DiscussionThread> {
+    private DiscussionThread thread;
+    private boolean read;
+
+    public ReadThreadTask(@NonNull Context context,
```

Comment:
`SetThreadReadTask` sounds a reasonable suggestion, since we're taking the read value as a parameter. I was following the naming pattern of the other similar tasks e.g. `VoteThreadTask` and `FlagThreadTask`, and I guess these should also be renamed similarly for the same reason. I'll add a new commit that makes these changes.